src/components/admin/content/content-management.jsx

```
@@ -1,4 +1,5 @@
-import { useState, useEffect } from 'react'
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

ESLint

[error] Definition for rule '@typescript-eslint/no-unused-vars' was not found.

(@typescript-eslint/no-unused-vars)